client/client_priv.h

```diff
@@ -120,6 +120,8 @@ enum options_client
   OPT_ENABLE_COMPRESSED_COLUMNS_WITH_DICTIONARIES,
   OPT_DROP_COMPRESSION_DICTIONARY,
   OPT_ORDER_BY_PRIMARY_DESC,
+  OPT_START_SQL_FILE,
```

Comment:
Should this be wrapped into `#ifndef DBUG_OFF` as well?

Reply:
This file is common for several tools from 'client' directory. I didn't investigate if they share some common parts which would cause enum values mismatch/shift when some of them built in debug and other in release. 
Oh the other size mysqldump does not use all these enums anyway (they are used by other client tools), enum acts as common place for collecting cmd line options identifiers.

It will be just unused enum value in debug build. We can add #ifdef here but in my opinion there will be no advantage.